packages/ui-kit/src/helpers/chartUtils.ts

```diff
@@ -0,0 +1,194 @@
+  LogarithmicScale,
+  PointElement,
+  TimeSeriesScale,
+  Tooltip,
+  Filler
+} from 'chart.js'
+import React from 'react'
+import type { ThemeTokenProps } from '../themes'
+
+export type ChartJSDefaultStyleProps = {
+  theme: ThemeTokenProps
+  globalChartConfigProps?: (chart: typeof Chart) => void
+}
+
+let isChartJSRegistered = false
+
+export const initChartJs = () => {
+  if (isChartJSRegistered) {
+    return
+  }
+
+  /**
+   * It registers only the modules that will be used
+   * in the context of a BarChart and LineChart so
+   * we reduce bundle weight
+   */
+
+  Chart.register(
+    LogarithmicScale,
+    Tooltip,
+    Colors,
+    PointElement,
+    BarElement,
+    LineElement,
+    BarController,
+    LineController,
+    TimeSeriesScale,
+    CategoryScale,
+    LinearScale,
+    Filler
+  )
+
+  isChartJSRegistered = true
+}
+
+/**
+ * Retrieves the computed pixel font size as a number from a CSS font-size value.
+ * This function dynamically creates an element, applies the provided font size,
+ * attaches it to the DOM to calculate the computed style, and then removes it.
+ * It is useful for converting relative font sizes (e.g., 'rem', 'em') to their
+ * pixel equivalent for calculations that require a numeric value.
+ *
+ * @param {React.CSSProperties['fontSize']} value - The font size in CSS units to be converted.
+ * @returns {number} The computed font size in pixels as a numeric value.
+ */
+export const getPixelFontSizeAsNumber = (value: React.CSSProperties['fontSize']) => {
+  const element = document.createElement('div')
+  element.style.fontSize = `${value}`
+  element.style.position = 'absolute'
+  element.style.visibility = 'hidden'
+  element.textContent = 'M'
+
+  document.body.appendChild(element)
+  const computedFontSize = window.getComputedStyle(element).fontSize
+  document.body.removeChild(element)
+
+  return parseFloat(computedFontSize)
+}
+
+/**
+ * Converts a HEX color code to an RGBA color code.
+ * The function accepts both short (3-digit) and full (6-digit) HEX codes.
+ * It also handles the conversion of opacity from a scale of 0 to 1 or 0 to 100 to an RGBA opacity value.
+ *
+ * @param {string} hexCode - The HEX color code to be converted.
+ * @param {number} opacity - The opacity value on a scale from 0 to 1 (default is 1). Values from 0 to 100 are also accepted for backward compatibility.
+ * @returns {string} The resulting RGBA color code as a string.
+ */
+export const convertHexToRGBA = (hexCode: string, opacity = 1) => {
+  let hex = hexCode.replace('#', '')
+
+  if (hex.length === 3) {
+    hex = `${hex[0]}${hex[0]}${hex[1]}${hex[1]}${hex[2]}${hex[2]}`
+  }
+
+  const r = parseInt(hex.substring(0, 2), 16)
+  const g = parseInt(hex.substring(2, 4), 16)
+  const b = parseInt(hex.substring(4, 6), 16)
+
+  /* Backward compatibility for whole number based opacity values. */
+  if (opacity > 1 && opacity <= 100) {
+    opacity = opacity / 100
+  }
+
+  return `rgba(${r},${g},${b},${opacity})`
+}
```

Comment:
Good candidate for a unit test, e.g., in `packages/ui-kit/src/helpers/chartUtils.test.ts`?

Reply:
Done!